docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml

```diff
@@ -0,0 +1,92 @@
+### YamlMime:FAQ
+metadata:
+  title: Nested app authentication and Outlook legacy tokens deprecation FAQ
+  description: Nested app authentication and Outlook legacy tokens deprecation FAQ
+  ms.service: microsoft-365
+  ms.subservice: add-ins
+  ms.topic: faq
+  ms.date: 04/08/2024
+  
+title: Nested app authentication and Outlook legacy tokens deprecation FAQ
+summary: |
+  Exchange user identity tokens and callback tokens will be turned off by default by October 2024. We recommend moving Outlook add-ins that use legacy Exchange tokens to nested app authentication.
+
+sections:
+  - name: General FAQ
+    questions:
+      - question: What is nested app authentication (NAA)?
+        answer: |
+          Nested app authentication enables single sign-on (SSO) for applications nested inside of supported Microsoft applications such as Outlook. Compared with existing full-trust authentication models, and the on-behalf-of flow, NAA provides better security and greater flexibility in app architecture, enabling the creation of rich, client-driven applications. For more information, see [Enable SSO in an Office Add-in using nested app authentication](enable-nested-app-authentication-in-your-add-in.md).
+
+      - question: What is the timeline for shutting down legacy tokens?
+        answer: |
+          After October 2024, Outlook add-ins that rely on legacy Exchange tokens will not work as expected since Exchange will no longer issue those legacy tokens.
```

Comment:
```suggestion
          After October 2024, Outlook add-ins that rely on legacy Exchange tokens will not work as expected since Exchange will no longer issue those legacy tokens by default.
```